cmd/start.go

```diff
@@ -25,9 +25,18 @@ var startCmd = &cobra.Command{
 	Long:  `Start the local environment by running all of the Extensions`,
 	Run: func(cmd *cobra.Command, args []string) {
 		forceFlag, _ := cmd.PersistentFlags().GetBool("force")
+		blackfireFlag, _ := cmd.PersistentFlags().GetBool("blackfire")
+
 		createLockFile(forceFlag)
 
 		for _, element := range cmd.Commands() {
+			if blackfireFlag && element.Use == "xdebug" {
```

Comment:
although this will stop the checking it wont actually stop the blackfire container from being started, because its in the main dm.yml. So i'm not 100% sure that it is necessary to even bother with this. I would only add the blackfire flag if you really want to prevent the container being started, and then you should implement the container in a separate compose file and start it separately as well.
Personally i would always start the blackfire container, but maybe just print a message about how to configure your container to use blackfire properly, dont enable xdebug in your site container and install the blackfire probe etc...... this might be a good long text message for the command ;)